.drone.yml

```diff
@@ -57,6 +57,19 @@ pipeline:
     commands:
```

Comment:
Please add to the `selenuim` settings

```
      - NODE_MAX_INSTANCES=5     
      - NODE_MAX_SESSION=5
```
See
https://github.com/openeuropa/oe_media/pull/163/files#diff-b54b39f1afced2465e1f3641db9d5bbf4f3a7fcf890996dfedd3c197bcb7f8c7R34